.github/workflows/run_test_suite.yml

```diff
@@ -0,0 +1,80 @@
+name: run-test-suite
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    services:
+      postgres:
+        image: postgres:10.8
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: test
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_ROOT_PASSWORD: password
+        ports:
+          - 3306
+        options: >-
+          --health-cmd="mysqladmin ping"
+          --health-interval=10s
+          --health-timeout=5s
+          --health-retries=3
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - ruby: 2.7
+            activerecord: 52
+          - ruby: 2.7
+            activerecord: 60
+          - ruby: 2.7
+            activerecord: 61
+          - ruby: 3.0
+            activerecord: 61
+    env:
+       PGUSER: postgres
+       PGPASSWORD: postgres
+       PGDATABASE: postgres
+       PGHOST: 127.0.0.1
+       PGPORT: 5432
+       MYSQL_PWD: password
+       MYSQL_HOST: 127.0.0.1
+       BUNDLE_GEMFILE: gemfiles/gemfile_${{ matrix.activerecord }}.gemfile
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup Ruby
+        # This will automatically get bug fixes and new Ruby versions (see https://github.com/ruby/setup-ruby#versioning)
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+          bundler-cache: true # runs 'bundle install' and caches installed gems automatically
+      - name: Install gems
+        run: |
+          bundle config path vendor/bundle
+          bundle install --jobs 4 --retry 3
+      - name: Setup mysql test database
+        run: |
+          sudo /etc/init.d/mysql start
+          mysql --port ${{ job.services.mysql.ports['3306'] }} -uroot -ppassword -e 'create database IF NOT EXISTS virtual_attributes_test;'
```

Comment:
password on the command line can be an issue here - think this is a FAQ to fix, but I don't have it.